arithmetization/src/main/java/net/consensys/linea/tracegeneration/rpc/RollupGenerateConflatedTracesToFileV0.java

```diff
@@ -121,52 +118,26 @@ private TraceService initTraceService() {
   }
 
   private String writeTraceToFile(final ZkTracer tracer, final String traceRuntimeVersion) {
-    final File file = generateOutputFile(traceRuntimeVersion);
-    final OutputStream outputStream = createOutputStream(file);
-
-    try (JsonGenerator jsonGenerator =
-        jsonFactory.createGenerator(outputStream, JsonEncoding.UTF8)) {
-      jsonGenerator.useDefaultPrettyPrinter();
-      jsonGenerator.writeObject(tracer.getJsonTrace());
-
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
-
-    return file.getAbsolutePath();
+    final Path fileName = generateOutputFileName(traceRuntimeVersion);
+    tracer.writeToFile(fileName);
+    return fileName.toAbsolutePath().toString();
   }
 
-  private OutputStream createOutputStream(final File file) {
-    try {
-      FileOutputStream fileOutputStream = new FileOutputStream(file);
-      if (isGzipEnabled) {
-        return new GZIPOutputStream(fileOutputStream);
-      }
-
-      return fileOutputStream;
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
-  }
-
-  private File generateOutputFile(final String tracesEngineVersion) {
-
+  private Path generateOutputFileName(final String tracesEngineVersion) {
     if (!Files.isDirectory(tracesPath) && !tracesPath.toFile().mkdirs()) {
       throw new RuntimeException(
           String.format(
               "Trace directory '%s' does not exist and could not be made.",
               tracesPath.toAbsolutePath()));
     }
 
-    return tracesPath
-        .resolve(
-            String.format(
-                "%.10s-%s.traces.%s",
-                System.currentTimeMillis(), tracesEngineVersion, getFileFormat()))
-        .toFile();
+    return Paths.get(
+        String.format(
+            "%.10s-%s.traces.%s",
+            System.currentTimeMillis(), tracesEngineVersion, getFileFormat()));
   }
 
   private String getFileFormat() {
-    return isGzipEnabled ? "json.gz" : "json";
+    return "lt";
```

Comment:
Is `lt` [this](https://www.analog.com/en/technical-articles/lt-programming-hex-file-format-documentation-in-circuit-programming.html) ?

Reply:
Nothing so deliberate, `lt` is **L**inea **T**trace :)